flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/test/program/SinkTestStep.java

```diff
@@ -138,6 +147,16 @@ public Builder consumedAfterRestore(String... expectedRows) {
             return this;
         }
 
+        public Builder testRawData() {
```

Comment:
rename to `changelogData()` because raw has too many meanings and this method should be a declaration.